app/forms/project-edit.tsx

```diff
@@ -23,29 +18,31 @@ import { getProjectSelector, useProjectSelector } from '~/hooks/use-params'
 import { addToast } from '~/stores/toast'
 import { pb } from '~/util/path-builder'
 
+const projectView = ({ project }: { project: string }) =>
+  apiq('projectView', { path: { project } })
```

Comment:
I'm realizing that these functions from selectors to query args are totally generic, likely to be used in multiple places, and could even be generated. In the meantime I'll try collecting some in a file.